tests/test_validate.py

```diff
@@ -102,35 +102,6 @@ def test_non_words(client):
     assert result == expected
 
 
-def test_non_words_POST(client):
-    """Just well-known not-a-word"""
-
-    params = {"response": "idk asdf lol n/a"}
-    resp = client.post("/validate", data=params)
-    expected = {
-        "bad_word_count": 4,
-        "common_word_count": 0,
-        "computation_time": 0.0015406608581542969,
-        "domain_word_count": 0,
-        "inner_product": -12.0,
-        "innovation_word_count": 0,
-        "processed_response": "nonsense_word nonsense_word nonsense_word n/a",
-        "remove_nonwords": True,
-        "remove_stopwords": True,
-        "response": "idk asdf lol n/a",
-        "spelling_correction": True,
-        "tag_numeric": False,
-        "uid": None,
-        "uid_found": False,
-        "valid": False,
-    }
-
-    result = resp.json
-    assert result["computation_time"] != 0
-    expected["computation_time"] = result["computation_time"]
-    assert result == expected
```

Comment:
Just making sure that I understand this -- we are making sure that given actual meets expected, but the one thing that will be noisy is the computation time.  To get around this, you just set the expected to actual so that the test will pass?

Reply:
right. That's also threshhold test to make sure the time computation did something (it's not zero) but I figure the other parts of the result would probably reflect something drastic going wrong.